spec/controllers/two_factor_authentication/recovery_code_verification_controller_spec.rb

```diff
@@ -55,6 +55,21 @@
         expect(response).to render_template(:show)
         expect(flash[:error]).to eq t('devise.two_factor_authentication.invalid_recovery_code')
       end
+
+      it 'tracks the max attempts event' do
+        properties = {
+          success?: false,
+          method: 'recovery code'
+        }
+
+        stub_analytics
+
+        expect(@analytics).to receive(:track_event).exactly(3).times.
+          with(Analytics::MULTI_FACTOR_AUTH, properties)
+        expect(@analytics).to receive(:track_event).with(Analytics::MULTI_FACTOR_AUTH_MAX_ATTEMPTS)
```

Comment:
should we specify a number of times for this to be clear that it's only happening once?